Formula/swiftly.rb

```diff
@@ -0,0 +1,39 @@
+class Swiftly < Formula
+  desc "A Swift toolchain installer and manager, written in Swift."
```

Comment:
fix "Description shouldn't start with an article." and "Description shouldn't end with a full stop."

```suggestion
  desc "Swift toolchain installer and manager"
```